### DISCUSSION TITLE
Declarative class names on development mode with Nextjs

### DISCUSSION THREAD

**Discussion by HashemKhalifa** [mantinedev/mantine#1170] [Feature requests]
Hello, 

Thanks a lot for the great effort!

I was wondering if there's a way to see a full class name on development mode?

<img width="407" alt="Screenshot 2022-04-03 at 05 30 16" src="https://user-images.githubusercontent.com/3355748/161410115-24e61fff-a92a-4396-b591-824996de4b44.png">

**Comment by rtivital** [comment]
It is the full classname in dev and prod mode.

**Reply by HashemKhalifa** [reply]
hey @rtivital, thank you for your quick response, 
Wondering why I'm getting the class names like the attached picture, is there's any config I need to change?

**Reply by rtivital** [reply]
These are classes that emotion generates, it cannot be changed.
